rubix-core/src/main/java/com/qubole/rubix/core/ReadRequestChain.java

```diff
@@ -28,7 +28,7 @@
  * <p>
  * One ReadRequestChain contains ReadRequests for same buffer
  */
-public abstract class ReadRequestChain implements Callable<Integer>
+public abstract class ReadRequestChain implements Callable<Long>
```

Comment:
can write a comment explaining why long is used instead of integer - since integer output is also correct for some RRCs.

Reply:
added